kessel-remediations.ksl

```diff
@@ -0,0 +1,124 @@
+// ========================================
+// Final v2 Permissions (Contingent)
+// ========================================
+
+// These create the final permissions that are actually checked
+// They require both the assigned permission and inventory access
+
+// Remediation permissions - require inventory_host_view as prerequisite
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_read_remediation_assigned', contingent:'remediations_read_remediation');
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_write_remediation_assigned', contingent:'remediations_write_remediation');
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_execute_remediation_assigned', contingent:'remediations_execute_remediation');
+
+// Playbook permissions - require inventory_host_view as prerequisite
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_read_playbook_assigned', contingent:'remediations_read_playbook');
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_write_playbook_assigned', contingent:'remediations_write_playbook');
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_execute_playbook_assigned', contingent:'remediations_execute_playbook');
+
+// System permissions - require inventory_host_view as prerequisite
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_read_system_assigned', contingent:'remediations_read_system');
+@rbac.add_contingent_permission(first:'inventory_host_view', second:'remediations_write_system_assigned', contingent:'remediations_write_system');
+
+// ========================================
+// Host-level Permission Exposure
+// ========================================
+
+// Expose workspace permissions to individual hosts for host-specific operations
+@hbi.expose_host_permission(v2_perm: 'remediations_read_remediation', host_perm: 'remediations_read_remediation');
+@hbi.expose_host_permission(v2_perm: 'remediations_write_remediation', host_perm: 'remediations_write_remediation');
+@hbi.expose_host_permission(v2_perm: 'remediations_execute_remediation', host_perm: 'remediations_execute_remediation');
+@hbi.expose_host_permission(v2_perm: 'remediations_read_playbook', host_perm: 'remediations_read_playbook');
+@hbi.expose_host_permission(v2_perm: 'remediations_write_playbook', host_perm: 'remediations_write_playbook');
+@hbi.expose_host_permission(v2_perm: 'remediations_execute_playbook', host_perm: 'remediations_execute_playbook');
+@hbi.expose_host_permission(v2_perm: 'remediations_read_system', host_perm: 'remediations_read_system');
+@hbi.expose_host_permission(v2_perm: 'remediations_write_system', host_perm: 'remediations_write_system');
+
+// ========================================
+// Wildcard Permission Mappings
+// ========================================
+
+// Handle wildcard permissions from v1 RBAC
```

Comment:
We shouldn't need these explicit wildcard mappings - add_v1_based_permission already creates the necessary wildcards and or's them in for the V2 permission. See rbac.ksl: https://github.com/RedHatInsights/rbac-config/blob/master/configs/stage/schemas/src/rbac.ksl#L53